src/components/About.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export default class About extends Component {
+  render() {
```

Comment:
In case of no states, it is simpler to implement as stateless components like NavBar and Video Viewer.